db/migrate/20150416085800_associate_parent_policies.rb

```diff
@@ -0,0 +1,16 @@
+class AssociateParentPolicies < ActiveRecord::Migration
+  POLICY_PARENT_MAPPINGS = {
+    'automatic-enrolment-in-workplace-pensions' => ['employment', 'older-people'],
+    'city-deal' => ['localism'],
+    'regional-growth-fund' => ['employment'],
+    'hs2-high-speed-rail' => ['rail-network']
```

Comment:
Are we certain enough that these slugs will be created?  Should we use content IDs instead?